### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by anukaal**
"TensorFlow supports Python 3.9 now"

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,14 +44,17 @@
         'requests',
         'tensorflow-federated',
     ],
-    # TensorFlow doesn't yet support Python 3.9.
-    python_requires='>=3.6, <3.9',
+    # TensorFlow support Python 3.9.
+    python_requires='>=3.6
```
Comment (kho) [inline setup.py:48]: I think you still need to close the string and add a comma to make this syntactically valid Python.

File: setup.py [hunk 2]
```diff
@@ -44,14 +44,17 @@
         'requests',
         'tensorflow-federated',
     ],
-    # TensorFlow doesn't yet support Python 3.9.
-    python_requires='>=3.6, <3.9',
+    # TensorFlow support Python 3.9.
```
Comment (kho) [inline setup.py:47]: Could you simply remove this comment as it is no longer necessary?

**Review by kho — CHANGES_REQUESTED** [review]
Thanks for taking the time to contribute!

Could you please:
(1) Revise the commit message to briefly mention why the update was done (e.g. "TensorFlow supports Python 3.9 now").
(2) Add 3.9 to this list https://github.com/google/fedjax/blob/fc8c0f693ec0da9a38ce041fe9bc76521d751604/.github/workflows/build_and_minimal_test.yml#L11 so we can make sure our code can be successfully installed under Python 3.9?

File: setup.py (hunk 1 above)
Comment (anukaal) [reply setup.py]: Done!

File: setup.py (hunk 2 above)
Comment (anukaal) [reply setup.py]: Done!

**Comment by anukaal** [comment]
> Thanks for taking the time to contribute!
> 
> Could you please:
> (1) Revise the commit message to briefly mention why the update was done (e.g. "TensorFlow supports Python 3.9 now").
> (2) Add 3.9 to this list
> 
> https://github.com/google/fedjax/blob/fc8c0f693ec0da9a38ce041fe9bc76521d751604/.github/workflows/build_and_minimal_test.yml#L11
> 
> so we can make sure our code can be successfully installed under Python 3.9?

Done !

File: setup.py [hunk 3]
```diff
@@ -44,14 +44,17 @@
         'requests',
         'tensorflow-federated',
     ],
-    # TensorFlow doesn't yet support Python 3.9.
-    python_requires='>=3.6, <3.9',
+
```
Comment (kho) [inline setup.py:47]: Could you remove this blank line?

File: setup.py [hunk 4]
```diff
@@ -44,14 +44,17 @@
         'requests',
         'tensorflow-federated',
     ],
-    # TensorFlow doesn't yet support Python 3.9.
-    python_requires='>=3.6, <3.9',
+   
+    python_requires='>=3.6'
```
Comment (kho) [inline setup.py:48]: I believe you need a comma at the end of this line.

**Review by kho — CHANGES_REQUESTED** [review]

File: setup.py (hunk 3 above)
Comment (anukaal) [reply setup.py]: Done!

File: setup.py (hunk 4 above)
Comment (anukaal) [reply setup.py]: Done!

**Review by kho — APPROVED** [review]

**Pull request merged**